QuarkusTools cannot find a virtual file over a WSL2 environment

My dev env:
- Win10 + WSL2 + Ubuntu 22.04.1 LTS
- IntelliJ 2022.3.2 Ultimate Edition
- GraalVM 21.3.0 (build 11.0.13+7-jvmci-21.3-b05)

Every time I open an aplication.properties file IntelliJ shows me an error related to the QuarkusTools plugin, as described below:

Internal error: java.lang.UnsupportedOperationException: Cannot find virtual file for 'file:////wsl$/Ubuntu-22.04/home/myusr/Workspaces/w1/projects/monitor-summary-api/src/main/resources/application.properties'

java.util.concurrent.CompletionException: java.lang.UnsupportedOperationException: Cannot find virtual file for 'file:////wsl$/Ubuntu-22.04/home/myusr/Workspaces/w1/projects/monitor-summary-api/src/main/resources/application.properties'
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:332)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:347)
	at java.base/java.util.concurrent.CompletableFuture$UniAccept.tryFire(CompletableFuture.java:708)
	at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)
	at java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2162)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$runAsBackground$2(QuarkusLanguageClient.java:124)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$runAsBackground$3(QuarkusLanguageClient.java:120)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$runAsBackground$4(QuarkusLanguageClient.java:130)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.UnsupportedOperationException: Cannot find virtual file for 'file:////wsl$/Ubuntu-22.04/home/myusr/Workspaces/w1/projects/monitor-summary-api/src/main/resources/application.properties'
	at com.redhat.devtools.intellij.lsp4mp4ij.psi.core.PropertiesManager.getMicroProfileProjectInfo(PropertiesManager.java:72)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$getProjectInfo$5(QuarkusLanguageClient.java:138)
	at com.redhat.devtools.intellij.quarkus.lsp.QuarkusLanguageClient.lambda$runAsBackground$2(QuarkusLanguageClient.java:122)
	... 16 more

--- a/src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/core/ls/PsiUtilsLSImpl.java
+++ b/src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/psi/internal/core/ls/PsiUtilsLSImpl.java
@@ -20,6 +20,7 @@ import com.intellij.openapi.util.TextRange;
 import com.intellij.openapi.vfs.VfsUtil;
 import com.intellij.openapi.vfs.VfsUtilCore;
 import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.openapi.vfs.VirtualFileManager;
 import com.intellij.psi.JavaPsiFacade;
 import com.intellij.psi.PsiClass;
 import com.intellij.psi.PsiDocumentManager;
@@ -129,7 +130,7 @@ public class PsiUtilsLSImpl implements IPsiUtils {
     @Override
     public VirtualFile findFile(String uri) throws IOException {
         //return LocalFileSystem.getInstance().refreshAndFindFileByIoFile(Paths.get(new URI(uri)).toFile());
-        return VfsUtil.findFileByURL(new URL(uri));
+        return VirtualFileManager.getInstance().findFileByUrl(VfsUtilCore.fixURLforIDEA(uri));
     }
 
     @Override

